.travis.yml

```diff
@@ -14,22 +14,19 @@ cache:
 
 matrix:
   include:
-    - php: 5.3
-      env: SYMFONY_VERSION='2.3.*'
-      dist: precise
     - php: 5.5
       env: SYMFONY_VERSION='2.7.*'
     - php: 5.6
       env: SYMFONY_VERSION='2.8.*'
-    - php: 7.0
-      env: DEPS=dev SYMFONY_VERSION='3.0.*'
     - php: 7.1
-      env: DEPS=dev SYMFONY_VERSION='3.0.*'
+      env: SYMFONY_VERSION='3.3.*'
+    - php: 7.1
+      env: DEPS=dev SYMFONY_VERSION='3.4.*'
+    - php: 7.1
+      env: DEVS=dev SYMFONY_VERSION='4.0.x-dev'
   allow_failures:
-    - php: 7.0
-      env: SYMFONY_VERSION='3.0.*'
     - php: 7.1
-      env: SYMFONY_VERSION='3.0.*'
+      env: SYMFONY_VERSION='4.0.x-dev'
```

Comment:
Maybe change this to `4.0.*`?

Reply:
Yep 👍

Reply:
Not here, these are the allowed failures